great-idea/css/index.css

```diff
@@ -63,4 +63,87 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.container {
+	width: 880px;
+	margin: 0 auto;
+}
+
+.navbar {
+	border: 1px solid red;
+}
+
+.navigation .logo {
+	display: flex;
+	justify-content: flex-end;
+}
+
+header .navigation {
+	display: flex;
+	justify-content: space-between;
+	margin-top: 15px;
+}
+
+.navigation a {
+	color: gray;
+	text-decoration: none;
+	font-size: 16px;
+	
+}
+
+.cta {
+	display: flex;
+	width: 880px;
+	justify-content: center; 
+	align-items: center;
+}
+
+.header-img {
+	padding: 25px 0;
+}
+
+.cta-header {
+	display: flex; 
+	justify-content: center;
+	text-align: center;  
+	font-size: 64px;
+}
```

Comment:
a change in flex direction here could have saved you the button margin a little lower down.